src/main/kotlin/no/nav/syfo/infrastructure/client/veiledertilgang/VeilederTilgangskontrollClient.kt

```diff
@@ -46,6 +49,44 @@ class VeilederTilgangskontrollClient(
         }
     }
 
+    suspend fun veilederPersonerAccess(
+        personidenter: List<PersonIdent>,
+        token: String,
+        callId: String,
+    ): List<String>? {
```

Comment:
Bør vi få med typen her?
```suggestion
    ): List<PersonIdent>? {
```

Reply:
APIet svarer med liste av strings, men jeg kan jo mappe det om i clienten så vi leverer `PersonIdent` videre inn i løsningen vår 😊